pkgs/development/python-modules/pykde4/default.nix

```diff
@@ -0,0 +1,49 @@
+{ kdelibs4, pythonPackages, kdepimlibs, shared_desktop_ontologies,
```

Comment:
can you list the actual dependencies instead of the package set

Reply:
@FRidh This way?